packages/server/lib/plugins/child/run_require_async_child.js

```diff
@@ -122,8 +124,15 @@ function run (ipc, file, projectRoot) {
         debug(`User doesn't have esbuild. Going to use native node imports.`)
 
         // We cannot replace the initial `require` with `await import` because
-        // Certain modules cannot be dynamically imported
-        return await import(file)
+        // Certain modules cannot be dynamically imported. If this throws, however, we want
+        // to show the original error that was thrown, because that's ultimately the source of the problem
+        try {
+          return await import(file)
+        } catch (e) {
+          // If we aren't able to import the file at all, throw the original error, since that has more accurate information
+          // of what failed to begin with
+          throw originalError
```

Comment:
Feels like a good thing to debug

```suggestion
          debug('esbuild fallback for loading config failed, throwing original error. esbuild error: %o', e)
          throw originalError
```